### ISSUE TITLE
BUG: Converting t3d challenge to hub, is failing to convert some parameters

### ISSUE THREAD

**Issue by Comoque1** [cherrytwist/Server#2057]
**Describe the bug**

**To Reproduce**
1. Go to Convert the challenge `t3d` to hub using the following mutation:
```
mutation ConvertChallengeToHub($convertData: ConvertChallengeToHubInput!) {
  convertChallengeToHub(convertData: $convertData) {
    displayName,
    id
  }
}

query variables:
{
  "convertData":
  {
    "challengeID": "t3d"
  }
}
```
2. The following data fails to be converted:
 - Tags on context for the new hub and challenges (they are missing)
![image.png](https://images.zenhubusercontent.com/5f5ff9fd50a1a94a1f25f32c/784e28f2-f1ef-48e0-90ee-a96f57c101bb)
 - opening an aspect doesn't load its data (this could be only, because the verification is performed locally and there is no link to matrix db)
![image.png](https://images.zenhubusercontent.com/5f5ff9fd50a1a94a1f25f32c/619e0d50-febd-42f4-afce-f5dd514e2ad0)
 - hub templates are default: not the configured ones (which may be OK)
![image.png](https://images.zenhubusercontent.com/5f5ff9fd50a1a94a1f25f32c/2134ff41-97ad-44e1-b299-a1740786384b)
 - preference are different for the new hub and challenges
 - old applications are not available
![image.png](https://images.zenhubusercontent.com/5f5ff9fd50a1a94a1f25f32c/15b6025d-034a-4d52-9836-faa8f62f635f)
 - old community admins are not admins anymore
![image.png](https://images.zenhubusercontent.com/5f5ff9fd50a1a94a1f25f32c/10197230-93ea-4b8b-916c-20462ecb26b1)

**Expected behavior**
TBD

**Comment by techsmyth** [comment]
Nice validation 👌

comments
- tags is a bug, surprised not picked up
- Aspects: this is due yo matrix being different. You should be able to verify the migration of the other data via graphql api without querying comments messages
- Hub templates as default is what it should be. There are no templates on challenges. 
- applications: ok these are not coming over, the hub is open anyway so not using them anymore
- Admins one is odd, maybe a bug. 

Overall  neither tags nor admin is a blocker as trivial to address after conversion happens so unless anything else shows up this is fit for purpose.

**Comment by Comoque1** [comment]
@techsmyth, forgot to mention:
 - to all new challenges `nameIds` after opportunity conversion is added a `c` char at the end. Not sure if this is done so for a purpose 
![image.png](https://images.zenhubusercontent.com/5f5ff9fd50a1a94a1f25f32c/1c980554-9e43-46e3-9beb-79717d0aba5c)
 - the aspect data is correctly returned from the server

**Comment by techsmyth** [comment]
Yes c character is by design but on reflection is not needed if done as part of challenge-hub. Again not a blocker but worth raising as a separate bug, or just convert this issue to deal with the “c”

**Comment by techsmyth** [comment]
No additional action to be taken on this for now.
